src/java/org/apache/cassandra/db/compaction/CompactionController.java

```diff
@@ -145,9 +147,9 @@ public static Set<SSTableReader> getFullyExpiredSSTables(ColumnFamilyStore cfSto
         if (compacting == null)
             return Collections.<SSTableReader>emptySet();
 
-        if (doNotPurgeTombstones(cfStore.keyspace.getName())) {
-            logger.debug("not looking for droppable sstables - doNotPurgeTombstones returned true for keyspace {}",
-                         SafeArg.of("keyspace", cfStore.keyspace.getName()));
+        if (doNotPurgeTombstones(cfStore))
+        {
+            logger.debug("Not looking for droppable sstables - doNotPurgeTombstones returned true for keyspace {}", cfStore.keyspace.getName());
```

Comment:
why remove the safearg?

Reply:
Mistake on the rebase deconflicting, nice catch!